charms/osm/sshproxy.py

```diff
@@ -245,215 +245,5 @@ def _ssh(self, cmd):
             stdout.read().decode("utf-8").strip(),
             stderr.read().decode("utf-8").strip(),
         )
+     )
```

Comment:
This needs to be remove?

Reply:
Also, if you can, add this in verify_credentials pydoc:

"""
Verify credentials

:return (bool, str): (Boolean ..., strderr)
"""